scripts/spec-coverage-check.sh

```
@@ -0,0 +1,238 @@
+done
+
+# ============================================================
+# CHECK: Tests pass (unless --skip-run)
+# ============================================================
+
+if [[ "$SKIP_RUN" == true ]]; then
+    check_skip "Test execution (--skip-run)"
+elif [[ ${#TEST_FILES[@]} -gt 0 && $MISSING -eq 0 ]]; then
+    RUN_FAILED=false
+    for test_file in "${TEST_FILES[@]}"; do
+        if ! npx vitest run "$REPO_ROOT/$test_file" 2>&1 >/dev/null; then
```

Shellcheck (0.11.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)